catalyst/core/runner.py

```diff
@@ -241,9 +242,8 @@ def _run_batch_train_step(self, batch: Mapping[str, Any]):
         self.state.output = self.forward(batch)
 
     @torch.no_grad()
-    def predict_batch(
-        self, batch: Mapping[str, Any], **kwargs
-    ) -> Mapping[str, Any]:
+    def predict_batch(self, batch: Mapping[str, Any],
+                      **kwargs) -> Mapping[str, Any]:
```

Comment:
codestyle please :)

Reply:
Well, this one was actually made by `yapf`, i.e `bash ./bin/yapf.sh --all-in-place`

Reply:
Probably my local codestyle scripts are not up to date?

Reply:
Actually, `yapf` restores all three your suggestions back to the current revision.

Reply:
please run `pip install -r ./requirements/requirements-dev.txt` for correct yapf version